README.md

```diff
@@ -1,6 +1,18 @@
-# [Octobox](https://octobox.io) &#128238;
+# [Octobox](https://octobox.io) &#128238; <small>Untangle your GitHub Notifications.<small>
 
-:postbox: Untangle your GitHub Notifications.
+Octobox helps you manage your GitHub notifications efficiently so you can spend less time managing and more time getting things done.
+
+- **Don't lose track** - Octobox adds an extra "archived" state to each notification so you can mark it as "done". If anything happens on an archived thread, issue or PR, Octobox will move it back into your inbox.
+
+- **Starred notifications** - Let's be honest, you probably don't have a 'favourite' issue. Octobox lets you highlight important notifications with a star so you can come back and find them easily.
```

Comment:
kinda think 

>**But** Octobox lets you highlight important notifications with a star so you can come back and find them easily.

Might be an easier read.

Reply:
👍